### DISCUSSION TITLE
Privatization deployment of community version

### DISCUSSION THREAD

**Discussion by eric-gitta-moore** [amplication/amplication#6669] [Q&A] [answered]
Does the community version provide a docker image that can be directly deployed?

Currently, I see in the source code that all images can be packaged with one click through `package: container`. May I ask if there is a `docker-compose. yml` file provided to deploy the image in one click?

Currently, we have seen `deploy: container` in `project.json`, but the commands option is not provided.

**Comment by overbit** [comment, marked as answer]
Already answered here: https://github.com/amplication/amplication/discussions/5986 

At the moment we don't provide any docker-compose for the deployment as it would need a different type of licence.
If you are interested, reach out to us here: https://amplication.com/contact-us
